multi-int-store/src/main/java/com/connexta/multiintstore/services/impl/IndexedMetadataDao.java

```diff
@@ -31,14 +30,7 @@ public IndexedMetadataDao(IndexedMetadataRepository repository) {
   }
 
   @Override
-  public void save(IndexedProductMetadata indexedProductMetadata)
```

Comment:
❓ Since all these commands mirror an IndexedMetadataRepository, should we just pass that around instead?